pages/dictionarypage/dictionary.js

```
@@ -147,17 +165,19 @@ function createNounSummaryTables() {
+        const genders = ["Exhalted", "Rational", "Monstrous", "Irrational", "Magical", "Mundane", "Abstract"];
         const numbers = ["Singular", "Dual", "Plural"];
 
         // Remove existing tables if they exist
-        ["dirSummaryTable", "recSummaryTable"].forEach(id => {
-            const oldTable = document.getElementById(id);
-            if (oldTable && oldTable.parentElement) {
-                oldTable.parentElement.remove();
-            }
-        });
-
+        if (["dirSummaryTable", "recSummaryTable"]) {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)